contracts/axelar-interchain-token-service/src/storage_types.rs

```diff
@@ -0,0 +1,17 @@
+use soroban_sdk::{contracttype, String};
+
+#[contracttype]
+#[derive(Clone, Debug)]
+pub struct TrustedItsAddress {
+    pub chain_name: String,
+    pub trusted_address: String,
+}
+
+#[contracttype]
+#[derive(Clone, Debug)]
+pub enum DataKey {
+    Initialized,
+    AuthModule,
+    Owner,
+    TrustedAddress(TrustedItsAddress),
```

Comment:
the key is the chain name, the value is the address. See `SignerHashByEpoch` in the gateway for reference